iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -125,6 +125,7 @@ public class HiveIcebergMetaHook implements HiveMetaHook {
   private static final List<org.apache.commons.lang3.tuple.Pair<Integer, byte[]>> EMPTY_FILTER =
       Lists.newArrayList(org.apache.commons.lang3.tuple.Pair.of(1, new byte[0]));
   static final String MIGRATED_TO_ICEBERG = "MIGRATED_TO_ICEBERG";
+  static final String ORC_FILES_ONLY = "orc.files.only";
```

Comment:
This is probably ends being an Iceberg-Hive integration logic only, and thus perhaps we should prepend this with "iceberg" so it's `iceberg.orc.files.only` or something similar.